components/sidebar/SearchView/HelpModal.tsx

```diff
@@ -41,28 +41,27 @@ const HelpModal = ({ isOpen, onClose }: props) => {
               <FormControl as="span">
                 <Text as="span" display="flex" alignItems="center">
                   Use the search function to search for anything in a course,
-                  including course code, substrings of course code, course
-                  title, or description.
+                  including the course code, substrings of the course code, course
+                  title, or even the description.
                 </Text>
                 <FormHelperText fontWeight={400}>
-                  For instance,
+                  For example:
                   <UnorderedList>
                     <ListItem>
-                      if you know the specific course code for a first year
+                      If you know the specific code for a first year
                       political science course you need to take, you can search
-                      for {'"pol101"'};
+                      for {'"POL101"'}.
                     </ListItem>
                     <ListItem>
-                      if you know you want to take a course in pseudoscience,
-                      you can search for {'"eco"'};
+                      If you want to take a course in pseudoscience,
+                      you can search for {'"ECO"'}.
                     </ListItem>
                     <ListItem>
-                      if you know you want to take a course about global
-                      affairs, you can search for {'"human rights"'};
+                      If you want to take a course about global
+                      affairs, you can search for {'"human rights"'}.
                     </ListItem>
                     <ListItem>
-                      if you know you want to take the easiest course at the
-                      university, you can search for {'"mat337"'}
+                      If you want to take the easiest course at UofT, you can search for {'"MAT337"'}
```

Comment:
As above.

Reply:
I've made the changes below, thank you for your insight.